src/scene/materials/standard-material.js

```diff
@@ -988,9 +997,10 @@ function _defineFlag(name, defaultValue) {
 function _defineMaterialProps() {
     _defineColor('ambient', new Color(0.7, 0.7, 0.7));
     _defineColor('diffuse', new Color(1, 1, 1));
-    _defineColor('specular', new Color(0, 0, 0));
+    _defineColor('specular', new Color(1, 1, 1));
```

Comment:
Wondering about this change: is the meaning/usage of specular now inverse of before? will this break existing projects?

Reply:
Okay, I think this is a good point. I will revert it to 0, but set specular tint to 1 when metalness is used. The problem here was that when metalness was enabled, specularity would be set to 0 and all specular effects would disappear because metalness enables specular, so if specular tint was the default it would just swallow all specular effects.